CMakeLists.txt

```diff
@@ -306,6 +314,15 @@ else()
 	endif (MSVC)
 endif()
 
+if (NOT MSVC)
+	if (abi STREQUAL "c++03")
+		add_definitions(-std=${abi})
```

Comment:
the CMakeLists.txt probably hasn't worked with anything other than gcc compatible and msvc compilers before either though.